src/client/reducers.js

```diff
@@ -134,6 +134,7 @@ export const getIsLoadingNotifications = state => fromUser.getIsLoadingNotificat
 export const getFetchFollowListError = state => fromUser.getFetchFollowListError(state.user);
 export const getLatestNotification = state => fromUser.getLatestNotification(state.user);
 export const getUserLocation = state => fromUser.getUserLocation(state.user);
+export const getAccuracyChartCondition = state => fromUser.getAccuracyChartCondition(state.user);
```

Comment:
rename 'getAccuracyChartCondition' to 'getAccuracyChartLoaded'

Reply:
+